mava/systems/jax/ippo/config.py

```diff
@@ -16,10 +16,19 @@
 """Default hyperparameters for IPPO system."""
 from dataclasses import dataclass
 
+from mava.components.jax import executing
+
 
 @dataclass
 class IPPODefaultConfig:
     sample_batch_size: int = 512
     sequence_length: int = 20
     period: int = 10
     use_next_extras: bool = False
+
+
+# The components that a needed for IPPO with a recurrent policy.
```

Comment:
Comment above but also, this would be better in the components.py file. Also, it should not be in an IPPO folder since the components are general, right?

Reply:
These are the configs for IPPO specifically. So it is not general. Each component also has its own general default configs.

Reply:
Should we not have these component blocks in components.py instead of config.py

Reply:
O yes. I completely agree :+1: I updated the code.

Reply:
Thanks